src/libostree/ostree-repo-private.h

```diff
@@ -153,6 +157,7 @@ struct OstreeRepo {
   guint64 tmp_expiry_seconds;
   gchar *collection_id;
   gboolean add_remotes_config_dir; /* Add new remotes in remotes.d dir */
+  gint lock_timeout;
```

Comment:
Maybe name it `lock_timeout_seconds` to make the units more obvious? Same for the `#define` above.

Reply:
Sure